# differential drive with POSQ extend function
extender: ClosedLoop
controller: POSQ
kinematicModel: DifferentialDrive
loopN: 1

# parameters
k_rho: 1
kv: 0.1
k_alpha: 2
k_phi: 0.1
